projects/elife.yaml

```diff
@@ -2099,3 +2099,17 @@ data-pipeline:
                  deletion-policy: delete
             "{instance}-elife-data-pipeline-archive":
                  deletion-policy: retain
+    gcp:
+        bigquery:
+            "{instance}": # dataset
+                # does 'project' belong beneath dataset? do we want an instance affecting multiple projects?
```

Comment:
I put datasets at the top level as there doesn't seem to be a resource to create that is spanning multiple datasets. The alternative is in fact to move them into a `datasets` key and have `project` and possibly other defaults right inside `bigquery`. We can probably change this later when we know more, it only propagates up to the `cfngen` module and then `context` hides this anyway.